docs/src/kani-result-types.md

```diff
@@ -0,0 +1,73 @@
+# Kani result types
+
+The result of a check in Kani can be one of the following:
+
+1. `SUCCESS`: This indicates that the check passed, i.e., the property holds.
+Note that in some cases, the property may hold _vacuously_. This can occur
+because the property is unreachable, or because the harness is
+_over-constrained_.
+
+Example:
+```rust
+{{#include getting-started/result-types/src/main.rs:success_example}}
+```
+The output from Kani indicates that the assertion holds:
+```
+Check 4: success_example.assertion.4
+         - Status: SUCCESS
+         - Description: "assertion failed: sum == 6"
+```
+
+2. `FAILURE`: This indicates that the check failed, i.e., the property doesn't
+hold. In this case, you can view the failure trace by re-running with the
+`--visualize` option.  This generates a `report` directory that contains the
+trace information. Load the `report/html/index.html` file in a browser to
```

Comment:
The new cargo-kani prints where the report was generated, and this is under `target/` for `cargo-kani` vs `kani`, so I recommend not stating the path here.

Reply:
Done.